theme/templates/base.html

```diff
@@ -78,31 +78,71 @@ <h2 class="text-white">sciwork</h2>
   {% block content %}{% endblock %}
 </main>
 
-<footer class="items-center bg-gray-800 text-white grow">
-  <div class="container mx-auto max-w-6xl px-5 py-2">
-    <h3>contact us</h3>
-    <div class="py-4 text-xl">
-    <a class="fa-stack" target="_blank" href="https://twitter.com/intent/tweet?screen_name=sciwork&ref_src=twsrc%5Etfw" data-show-count="false">
-      <i class="fas fa-circle fa-stack-2x text-white"></i>
-      <i class="fab fa-twitter fa-stack-1x text-gray-800" style="padding: 0.1rem 0 0 0.1rem"></i>
-    </a>
-    <a class="fa-stack" target="_blank" href="mailto: contact@sciwork.dev">
-      <i class="fas fa-circle fa-stack-2x text-white"></i>
-      <i class="fas fa-envelope fa-stack-1x grey-text text-gray-800"></i>
-    </a>
-    <a class="fa-stack" target="_blank" href="https://github.com/sciwork/swportal">
-      <i class="fas fa-circle fa-stack-2x text-white"></i>
-      <i class="fab fa-github fa-stack-1x grey-text text-gray-800"></i>
-    </a>
+
+<footer class="bg-blue-900 text-white">
+  <div class="container mx-auto max-w-6xl px-5 py-2 mt-4 mb-4" >
+    <div class="flex flex-wrap justify-between">
+      <div class="flex items-start">
+        <img class="inline h-12 w-12 mr-8 mt-3" src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/images/android-chrome-192x192.png"/>
+        <h2>sciwork</h2>
     </div>
-  </div>
 
-  <div class="bg-gray-900">
-    <div class="container mx-auto max-w-6xl px-5 py-2">
-      the website is powered by
-      <a class="text-white" target="_blank" href="https://getpelican.com">Pelican</a>
-      and
-      <a class="text-white" target="_blank" href="https://python.org">Python</a>.
+      <div class="flex flex-col">
+        <h3 class="text-red-800 mb-2">activity</h3>
```

Comment:
Looks like mb-2 does not have css definition:
![CleanShot 2024-08-25 at 21 02 37@2x](https://github.com/user-attachments/assets/9aa8a17c-edef-4e70-83e7-f72d86f60f99)